Aug 31, 2010 9:31:25 AM org.apache.catalina.core.StandardWrapperValve invoke
SEVERE: Servlet.service() for servlet action threw exception
com.jcorporate.expresso.core.db.DBException: com.jcorporate.expresso.core.db.DBConnection.connect(String, String):Cant get connection to database via driver 'org.firebirdsql.jdbc.FBDriver' and URL 'jdbc:firebirdsql:localhost/3050:E:/WORKING/PROJECTS/Henessy/Database/SSP_HRM_DB_30.GDB?lc_ctype=UNICODE_FSS' (no description):GDS Exception. 335544721. Unable to complete network request to host "localhost".
Reason: Unable to complete network request to host "localhost".
	at com.jcorporate.expresso.core.db.DBConnection.connect(DBConnection.java:742)
	at com.jcorporate.expresso.core.db.DBConnectionPool.buildNewConnection(DBConnectionPool.java:406)
	at com.jcorporate.expresso.core.db.DBConnectionPool.createNewConnection(DBConnectionPool.java:456)
	at com.jcorporate.expresso.core.db.DBConnectionPool.getConnection(DBConnectionPool.java:1039)
	at com.jcorporate.expresso.core.db.DBConnectionPool.getConnection(DBConnectionPool.java:1154)
	at com.jcorporate.expresso.core.dataobjects.jdbc.JDBCExecutor.retrieve(JDBCExecutor.java:821)
	at com.jcorporate.expresso.core.dbobj.DBObject.retrieve(DBObject.java:4961)
	at com.jcorporate.expresso.core.dbobj.SecuredDBObject.retrieve(SecuredDBObject.java:609)
	at com.jcorporate.expresso.core.dbobj.DBObject.find(DBObject.java:2247)
	at com.jcorporate.expresso.core.dbobj.SecuredDBObject.find(SecuredDBObject.java:370)
	at com.jcorporate.expresso.services.dbobj.DefaultUserInfo.find(DefaultUserInfo.java:1359)
	at com.jcorporate.expresso.core.security.User.find(User.java:395)
	at com.jcorporate.expresso.core.servlet.CheckLogin.check(CheckLogin.java:275)
	at com.jcorporate.expresso.core.servlet.CheckLogin.checkLogin(CheckLogin.java:156)
	at com.jcorporate.expresso.core.controller.Controller.execute(Controller.java:1675)
	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:507)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:690)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:803)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:175)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:844)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Thread.java:619)
com.jcorporate.expresso.core.dataobjects.DataException: com.jcorporate.expresso.core.dataobjects.jdbc.JDBCExecutor.retrieve(DataObject) error
	at com.jcorporate.expresso.core.dataobjects.jdbc.JDBCExecutor.retrieve(JDBCExecutor.java:952)
	at com.jcorporate.expresso.core.dbobj.DBObject.retrieve(DBObject.java:4961)
	at com.jcorporate.expresso.core.dbobj.SecuredDBObject.retrieve(SecuredDBObject.java:609)
	at com.jcorporate.expresso.core.dbobj.DBObject.find(DBObject.java:2247)
	at com.jcorporate.expresso.core.dbobj.SecuredDBObject.find(SecuredDBObject.java:370)
	at com.jcorporate.expresso.services.dbobj.DefaultUserInfo.find(DefaultUserInfo.java:1359)
	at com.jcorporate.expresso.core.security.User.find(User.java:395)
	at com.jcorporate.expresso.core.servlet.CheckLogin.check(CheckLogin.java:275)
	at com.jcorporate.expresso.core.servlet.CheckLogin.checkLogin(CheckLogin.java:156)
	at com.jcorporate.expresso.core.controller.Controller.execute(Controller.java:1675)
	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:507)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:690)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:803)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:175)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:844)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Thread.java:619)
com.jcorporate.expresso.core.controller.NonHandleableException: Nested exception: DataException: com.jcorporate.expresso.core.dataobjects.jdbc.JDBCExecutor.retrieve(DataObject) error
	at com.jcorporate.expresso.core.servlet.CheckLogin.check(CheckLogin.java:303)
	at com.jcorporate.expresso.core.servlet.CheckLogin.checkLogin(CheckLogin.java:156)
	at com.jcorporate.expresso.core.controller.Controller.execute(Controller.java:1675)
	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:507)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:690)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:803)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:175)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:844)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Thread.java:619)
Aug 31, 2010 9:32:44 AM org.apache.catalina.core.StandardWrapperValve invoke
SEVERE: Servlet.service() for servlet action threw exception
com.jcorporate.expresso.core.db.DBException: com.jcorporate.expresso.core.db.DBConnection.connect(String, String):Cant get connection to database via driver 'org.firebirdsql.jdbc.FBDriver' and URL 'jdbc:firebirdsql:localhost/3050:E:/WORKING/PROJECTS/Henessy/Database/SSP_HRM_DB_30.GDB?lc_ctype=UNICODE_FSS' (no description):GDS Exception. 335544721. Unable to complete network request to host "localhost".
Reason: Unable to complete network request to host "localhost".
	at com.jcorporate.expresso.core.db.DBConnection.connect(DBConnection.java:742)
	at com.jcorporate.expresso.core.db.DBConnectionPool.buildNewConnection(DBConnectionPool.java:406)
	at com.jcorporate.expresso.core.db.DBConnectionPool.createNewConnection(DBConnectionPool.java:456)
	at com.jcorporate.expresso.core.db.DBConnectionPool.getConnection(DBConnectionPool.java:1039)
	at com.jcorporate.expresso.core.db.DBConnectionPool.getConnection(DBConnectionPool.java:1154)
	at com.jcorporate.expresso.core.dataobjects.jdbc.JDBCExecutor.retrieve(JDBCExecutor.java:821)
	at com.jcorporate.expresso.core.dbobj.DBObject.retrieve(DBObject.java:4961)
	at com.jcorporate.expresso.core.dbobj.SecuredDBObject.retrieve(SecuredDBObject.java:609)
	at com.jcorporate.expresso.core.dbobj.DBObject.find(DBObject.java:2247)
	at com.jcorporate.expresso.core.dbobj.SecuredDBObject.find(SecuredDBObject.java:370)
	at com.jcorporate.expresso.services.dbobj.DefaultUserInfo.find(DefaultUserInfo.java:1359)
	at com.jcorporate.expresso.core.security.User.find(User.java:395)
	at com.jcorporate.expresso.core.servlet.CheckLogin.check(CheckLogin.java:275)
	at com.jcorporate.expresso.core.servlet.CheckLogin.checkLogin(CheckLogin.java:156)
	at com.jcorporate.expresso.core.controller.Controller.execute(Controller.java:1675)
	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:507)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:690)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:803)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:175)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:844)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Thread.java:619)
com.jcorporate.expresso.core.dataobjects.DataException: com.jcorporate.expresso.core.dataobjects.jdbc.JDBCExecutor.retrieve(DataObject) error
	at com.jcorporate.expresso.core.dataobjects.jdbc.JDBCExecutor.retrieve(JDBCExecutor.java:952)
	at com.jcorporate.expresso.core.dbobj.DBObject.retrieve(DBObject.java:4961)
	at com.jcorporate.expresso.core.dbobj.SecuredDBObject.retrieve(SecuredDBObject.java:609)
	at com.jcorporate.expresso.core.dbobj.DBObject.find(DBObject.java:2247)
	at com.jcorporate.expresso.core.dbobj.SecuredDBObject.find(SecuredDBObject.java:370)
	at com.jcorporate.expresso.services.dbobj.DefaultUserInfo.find(DefaultUserInfo.java:1359)
	at com.jcorporate.expresso.core.security.User.find(User.java:395)
	at com.jcorporate.expresso.core.servlet.CheckLogin.check(CheckLogin.java:275)
	at com.jcorporate.expresso.core.servlet.CheckLogin.checkLogin(CheckLogin.java:156)
	at com.jcorporate.expresso.core.controller.Controller.execute(Controller.java:1675)
	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:507)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:690)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:803)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:175)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:844)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Thread.java:619)
com.jcorporate.expresso.core.controller.NonHandleableException: Nested exception: DataException: com.jcorporate.expresso.core.dataobjects.jdbc.JDBCExecutor.retrieve(DataObject) error
	at com.jcorporate.expresso.core.servlet.CheckLogin.check(CheckLogin.java:303)
	at com.jcorporate.expresso.core.servlet.CheckLogin.checkLogin(CheckLogin.java:156)
	at com.jcorporate.expresso.core.controller.Controller.execute(Controller.java:1675)
	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:507)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:690)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:803)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:175)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:844)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Thread.java:619)
Aug 31, 2010 9:33:08 AM org.apache.catalina.core.StandardWrapperValve invoke
SEVERE: Servlet.service() for servlet action threw exception
com.jcorporate.expresso.core.db.DBException: com.jcorporate.expresso.core.db.DBConnection.connect(String, String):Cant get connection to database via driver 'org.firebirdsql.jdbc.FBDriver' and URL 'jdbc:firebirdsql:localhost/3050:E:/WORKING/PROJECTS/Henessy/Database/SSP_HRM_DB_30.GDB?lc_ctype=UNICODE_FSS' (no description):GDS Exception. 335544721. Unable to complete network request to host "localhost".
Reason: Unable to complete network request to host "localhost".
	at com.jcorporate.expresso.core.db.DBConnection.connect(DBConnection.java:742)
	at com.jcorporate.expresso.core.db.DBConnectionPool.buildNewConnection(DBConnectionPool.java:406)
	at com.jcorporate.expresso.core.db.DBConnectionPool.createNewConnection(DBConnectionPool.java:456)
	at com.jcorporate.expresso.core.db.DBConnectionPool.getConnection(DBConnectionPool.java:1039)
	at com.jcorporate.expresso.core.db.DBConnectionPool.getConnection(DBConnectionPool.java:1154)
	at com.jcorporate.expresso.core.dataobjects.jdbc.JDBCExecutor.retrieve(JDBCExecutor.java:821)
	at com.jcorporate.expresso.core.dbobj.DBObject.retrieve(DBObject.java:4961)
	at com.jcorporate.expresso.core.dbobj.SecuredDBObject.retrieve(SecuredDBObject.java:609)
	at com.jcorporate.expresso.core.dbobj.DBObject.find(DBObject.java:2247)
	at com.jcorporate.expresso.core.dbobj.SecuredDBObject.find(SecuredDBObject.java:370)
	at com.jcorporate.expresso.services.dbobj.DefaultUserInfo.find(DefaultUserInfo.java:1359)
	at com.jcorporate.expresso.core.security.User.find(User.java:395)
	at com.jcorporate.expresso.core.servlet.CheckLogin.check(CheckLogin.java:275)
	at com.jcorporate.expresso.core.servlet.CheckLogin.checkLogin(CheckLogin.java:156)
	at com.jcorporate.expresso.core.controller.Controller.execute(Controller.java:1675)
	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:507)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:690)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:803)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:175)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:844)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Thread.java:619)
com.jcorporate.expresso.core.dataobjects.DataException: com.jcorporate.expresso.core.dataobjects.jdbc.JDBCExecutor.retrieve(DataObject) error
	at com.jcorporate.expresso.core.dataobjects.jdbc.JDBCExecutor.retrieve(JDBCExecutor.java:952)
	at com.jcorporate.expresso.core.dbobj.DBObject.retrieve(DBObject.java:4961)
	at com.jcorporate.expresso.core.dbobj.SecuredDBObject.retrieve(SecuredDBObject.java:609)
	at com.jcorporate.expresso.core.dbobj.DBObject.find(DBObject.java:2247)
	at com.jcorporate.expresso.core.dbobj.SecuredDBObject.find(SecuredDBObject.java:370)
	at com.jcorporate.expresso.services.dbobj.DefaultUserInfo.find(DefaultUserInfo.java:1359)
	at com.jcorporate.expresso.core.security.User.find(User.java:395)
	at com.jcorporate.expresso.core.servlet.CheckLogin.check(CheckLogin.java:275)
	at com.jcorporate.expresso.core.servlet.CheckLogin.checkLogin(CheckLogin.java:156)
	at com.jcorporate.expresso.core.controller.Controller.execute(Controller.java:1675)
	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:507)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:690)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:803)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:175)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:844)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Thread.java:619)
com.jcorporate.expresso.core.controller.NonHandleableException: Nested exception: DataException: com.jcorporate.expresso.core.dataobjects.jdbc.JDBCExecutor.retrieve(DataObject) error
	at com.jcorporate.expresso.core.servlet.CheckLogin.check(CheckLogin.java:303)
	at com.jcorporate.expresso.core.servlet.CheckLogin.checkLogin(CheckLogin.java:156)
	at com.jcorporate.expresso.core.controller.Controller.execute(Controller.java:1675)
	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:507)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:690)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:803)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:175)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:844)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Thread.java:619)
Aug 31, 2010 9:34:06 AM org.apache.catalina.core.StandardWrapperValve invoke
SEVERE: Servlet.service() for servlet action threw exception
com.jcorporate.expresso.core.db.DBException: com.jcorporate.expresso.core.db.DBConnection.connect(String, String):Cant get connection to database via driver 'org.firebirdsql.jdbc.FBDriver' and URL 'jdbc:firebirdsql:localhost/3050:E:/WORKING/PROJECTS/Henessy/Database/SSP_HRM_DB_30.GDB?lc_ctype=UNICODE_FSS' (no description):GDS Exception. 335544721. Unable to complete network request to host "localhost".
Reason: Unable to complete network request to host "localhost".
	at com.jcorporate.expresso.core.db.DBConnection.connect(DBConnection.java:742)
	at com.jcorporate.expresso.core.db.DBConnectionPool.buildNewConnection(DBConnectionPool.java:406)
	at com.jcorporate.expresso.core.db.DBConnectionPool.createNewConnection(DBConnectionPool.java:456)
	at com.jcorporate.expresso.core.db.DBConnectionPool.getConnection(DBConnectionPool.java:1039)
	at com.jcorporate.expresso.core.db.DBConnectionPool.getConnection(DBConnectionPool.java:1154)
	at com.jcorporate.expresso.core.dataobjects.jdbc.JDBCExecutor.retrieve(JDBCExecutor.java:821)
	at com.jcorporate.expresso.core.dbobj.DBObject.retrieve(DBObject.java:4961)
	at com.jcorporate.expresso.core.dbobj.SecuredDBObject.retrieve(SecuredDBObject.java:609)
	at com.jcorporate.expresso.core.dbobj.DBObject.find(DBObject.java:2247)
	at com.jcorporate.expresso.core.dbobj.SecuredDBObject.find(SecuredDBObject.java:370)
	at com.jcorporate.expresso.services.dbobj.DefaultUserInfo.find(DefaultUserInfo.java:1359)
	at com.jcorporate.expresso.core.security.User.find(User.java:395)
	at com.jcorporate.expresso.core.servlet.CheckLogin.check(CheckLogin.java:275)
	at com.jcorporate.expresso.core.servlet.CheckLogin.checkLogin(CheckLogin.java:156)
	at com.jcorporate.expresso.core.controller.Controller.execute(Controller.java:1675)
	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:507)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:690)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:803)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:175)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:844)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Thread.java:619)
com.jcorporate.expresso.core.dataobjects.DataException: com.jcorporate.expresso.core.dataobjects.jdbc.JDBCExecutor.retrieve(DataObject) error
	at com.jcorporate.expresso.core.dataobjects.jdbc.JDBCExecutor.retrieve(JDBCExecutor.java:952)
	at com.jcorporate.expresso.core.dbobj.DBObject.retrieve(DBObject.java:4961)
	at com.jcorporate.expresso.core.dbobj.SecuredDBObject.retrieve(SecuredDBObject.java:609)
	at com.jcorporate.expresso.core.dbobj.DBObject.find(DBObject.java:2247)
	at com.jcorporate.expresso.core.dbobj.SecuredDBObject.find(SecuredDBObject.java:370)
	at com.jcorporate.expresso.services.dbobj.DefaultUserInfo.find(DefaultUserInfo.java:1359)
	at com.jcorporate.expresso.core.security.User.find(User.java:395)
	at com.jcorporate.expresso.core.servlet.CheckLogin.check(CheckLogin.java:275)
	at com.jcorporate.expresso.core.servlet.CheckLogin.checkLogin(CheckLogin.java:156)
	at com.jcorporate.expresso.core.controller.Controller.execute(Controller.java:1675)
	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:507)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:690)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:803)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:175)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:844)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Thread.java:619)
com.jcorporate.expresso.core.controller.NonHandleableException: Nested exception: DataException: com.jcorporate.expresso.core.dataobjects.jdbc.JDBCExecutor.retrieve(DataObject) error
	at com.jcorporate.expresso.core.servlet.CheckLogin.check(CheckLogin.java:303)
	at com.jcorporate.expresso.core.servlet.CheckLogin.checkLogin(CheckLogin.java:156)
	at com.jcorporate.expresso.core.controller.Controller.execute(Controller.java:1675)
	at org.apache.struts.action.RequestProcessor.processActionPerform(RequestProcessor.java:484)
	at org.apache.struts.action.RequestProcessor.process(RequestProcessor.java:274)
	at org.apache.struts.action.ActionServlet.process(ActionServlet.java:1482)
	at org.apache.struts.action.ActionServlet.doGet(ActionServlet.java:507)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:690)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:803)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:175)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:844)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Thread.java:619)
